docs/devel/DEVELOPMENT.md

```diff
@@ -149,20 +149,19 @@ files in the root of this repository:
 OPA uses Github Actions defined in the [.github/workflows](../../.github/workflows)
 directory.
 
-## Required Secrets
+## Github Action Secrets
 
-The following secrets are assumed to be configured for the Repository. Any fork of
-OPA will need to have them configured to be able to run the full CI workflow.
+The following secrets are used by the Github Action workflows:
 
 | Name | Description |
 |------|-------------|
-| S3_RELEASE_BUCKET | AWS S3 Bucket name to upload `edge` release binaries to. |
-| AWS_ACCESS_KEY_ID | AWS credentials required to upload to the configured `S3_RELEASE_BUCKET`. |
-| AWS_SECRET_ACCESS_KEY | AWS credentials required to upload to the configured `S3_RELEASE_BUCKET`. |
-| DOCKER_IMAGE | Full docker image name (with org) to tag and publish OPA images. |
-| DOCKER_WASM_BUILDER_IMAGE | Full docker image name (with org) to tag and publish WASM builder images. |
-| DOCKER_USER | Docker username for uploading release images. Will be used with `docker login` |
-| DOCKER_PASSWORD | Docker password or API token for the configured `DOCKER_USER`. Will be used with `docker login` |
+| S3_RELEASE_BUCKET | AWS S3 Bucket name to upload `edge` release binaries to. Optional -- If not provided the release upload steps are skipped. |
+| AWS_ACCESS_KEY_ID | AWS credentials required to upload to the configured `S3_RELEASE_BUCKET`. Optional -- If not provided the release upload steps are skipped. |
+| AWS_SECRET_ACCESS_KEY | AWS credentials required to upload to the configured `S3_RELEASE_BUCKET`. Optional -- If not provided the release upload steps are skipped. |
+| DOCKER_IMAGE | Full docker image name (with org) to tag and publish OPA images. Optional -- If not provided the image push steps are skipped. |
+| DOCKER_WASM_BUILDER_IMAGE | Full docker image name (with org) to tag and publish WASM builder images. Optional -- If not provided the image push steps are skipped. |
```

Comment:
I just noticed these secrets (DOCKER_IMAGE and DOCKER_WASM_BUILDER_IMAGE) are NOT referenced in the changes. Is the note about them being optional accurate? It looks like the conditionals are based on the credentials.

Reply:
Oops, they were victims of aggressive copy pasta. Those should be noted as option, and the like standard opa image names will be used if they are not provided.

Reply:
Updated the details for those two